src/core/src/UsdUtil.cpp

```diff
@@ -294,32 +294,11 @@ pxr::CesiumGeoreference defineCesiumGeoreference(const pxr::SdfPath& path) {
     return georeference;
 }
 
-pxr::CesiumTilesetAPI defineCesiumTileset(const pxr::SdfPath& path) {
+pxr::CesiumTileset defineCesiumTileset(const pxr::SdfPath& path) {
     auto stage = getUsdStage();
-    auto xform = pxr::UsdGeomXform::Define(stage, path);
-    assert(xform.GetPrim().IsValid());
+    auto tileset = pxr::CesiumTileset::Define(stage, path);
 
-    auto tileset = pxr::CesiumTilesetAPI::Apply(xform.GetPrim());
     assert(tileset.GetPrim().IsValid());
-
-    tileset.CreateUrlAttr();
-    tileset.CreateIonAssetIdAttr();
-    tileset.CreateIonAccessTokenAttr();
-    tileset.CreateMaximumScreenSpaceErrorAttr(pxr::VtValue(16.0f));
-    tileset.CreatePreloadAncestorsAttr(pxr::VtValue(true));
-    tileset.CreatePreloadSiblingsAttr(pxr::VtValue(true));
-    tileset.CreateForbidHolesAttr(pxr::VtValue(false));
-    tileset.CreateMaximumSimultaneousTileLoadsAttr(pxr::VtValue(uint32_t(20)));
-    tileset.CreateMaximumCachedBytesAttr(pxr::VtValue(uint64_t(536870912)));
-    tileset.CreateLoadingDescendantLimitAttr(pxr::VtValue(uint32_t(20)));
-    tileset.CreateEnableFrustumCullingAttr(pxr::VtValue(true));
-    tileset.CreateEnableFogCullingAttr(pxr::VtValue(true));
-    tileset.CreateEnforceCulledScreenSpaceErrorAttr(pxr::VtValue(true));
-    tileset.CreateCulledScreenSpaceErrorAttr(pxr::VtValue(64.0f));
-    tileset.CreateSuspendUpdateAttr(pxr::VtValue(false));
-    tileset.CreateSmoothNormalsAttr(pxr::VtValue(false));
-    tileset.CreateShowCreditsOnScreenAttr(pxr::VtValue(false));
```

Comment:
Related to the discussion in https://github.com/CesiumGS/cesium-omniverse/pull/339#discussion_r1228761878...

I think these lines should be added back unless they're a reason to remove them.

Reply:
I don't think the lines above have any effect, except for the possibility of altering default values. These attributes are created and default values set in the schema file. For example, if we look at [MaximumScreenSpaceError](https://github.com/CesiumGS/cesium-omniverse/pull/352#discussion_r1234055441), then to keep the code in sync we'd need to make sure the default in the schema matches the C++. If the C++ differs, then the UI will show the default value has changed and enable to button to reset the value.

Reply:
Ah, makes sense. I don't think we have any reason to set different default values in C++.

One thing I noticed is that creating attributes explicitly causes them to be saved in the .usda, whereas the other approach only saves values that are set. I would guess the second approach is the more USD way to do things.

**Create Attributes**
```
def Xform "Cesium_Tileset" (
    prepend apiSchemas = ["CesiumTilesetSchemaAPI"]
)
{
    float cesium:culledScreenSpaceError = 64
    bool cesium:enableFogCulling = 1
    bool cesium:enableFrustumCulling = 1
    bool cesium:enforceCulledScreenSpaceError = 1
    bool cesium:forbidHoles = 0
    prepend rel cesium:georeferenceBinding = </CesiumGeoreference>
    string cesium:ionAccessToken = ""
    int64 cesium:ionAssetId = 1
    uint cesium:loadingDescendantLimit = 20
    uint64 cesium:maximumCachedBytes = 536870912
    float cesium:maximumScreenSpaceError = 16
    uint cesium:maximumSimultaneousTileLoads = 20
    bool cesium:preloadAncestors = 1
    bool cesium:preloadSiblings = 1
    bool cesium:showCreditsOnScreen = 0
    bool cesium:smoothNormals = 0
    uniform token cesium:sourceType = "ion"
    bool cesium:suspendUpdate = 0
    string cesium:url
}
```

**Don't Create Attributes**
```
def Xform "Cesium_Tileset" (
    prepend apiSchemas = ["CesiumTilesetSchemaAPI"]
)
{
    prepend rel cesium:georeferenceBinding = </CesiumGeoreference>
    string cesium:ionAccessToken = ""
    int64 cesium:ionAssetId = 1
}
```

Reply:
Good call. We should. How about doing it as a small PR right after this? It seems like it would keep the commit log tidy.

Reply:
>One thing I noticed is that creating attributes explicitly causes them to be saved in the .usda, whereas the other approach only saves values that are set. 

Interesting. Then by not creating attributes in C++ we have the possibility of altering some values that would still auto-set on old USD files, like cache size or something. Even if we go the other route and keep the calls, it's probably more correct to get and set the attributes, since they're already created. We can move this discussion to the new PR.